packages/angular/package.json

```diff
@@ -55,13 +55,12 @@
     "files": [
       "test/**/*"
     ],
-    "compileEnhancements": false,
     "extensions": [
       "ts"
     ],
     "require": [
       "./test-config/ts-node.config.js",
-      "source-map-support/register"
+      "@cspotcode/source-map-support/register.js"
```

Comment:
Why do we switch to this fork? Is this a mandatory change?

Reply:
Good point. Initially something didn't work in relation to source-map-support and I thought because it is outdated. Turned out, only the `.js` ending was missing. I now reverted back to the original `original-source-map` and only set its dep to the latest version (only a patch version upgrade).

Reply:
I also checked that reverting to original-source-map does not increase the vulnerability count again